src/SUMMARY.md

```diff
@@ -52,6 +52,7 @@
     - [miri const evaluator](./miri.md)
 - [Parameter Environments](./param_env.md)
 - [Generating LLVM IR](./trans.md)
+- [Debugging the Compiler](./compiler-debugging.md)
```

Comment:
I think this should be moved after the testing chapter (ch 4)

Reply:
Done!